DecipherNow/gm-fabric-dashboard
# 1383 with url threadsgrid

Resolves #1383 

## Proposed Changes

  - Modify the API for the URL State HOC
  - Make the Threads View router driven
  
## How to Test

  - Start app
  - Select a JVM microservice
  - Select an Instance
  - Open Threads View
  - Use the Table Header controls to search, group, and sort
  - Confirm the URL changes
  - Hard Refresh after each control change to confirm it works as expected.
